src/service/blockChainManager.ts

```diff
@@ -16,4 +18,8 @@ export const createABlockChain = (arrayLength:number):block[] => new Array(array
   .fill(0)
   .map(createBlock);
 
-export const getLastBlock = (blockchain) => blockchain[blockchain.length - 1];
+export const getLastBlock = (blockchain:block[]) => blockchain[blockchain.length - 1];
+
+const integrityOfABlockCondition = (_block: any, index: number, blockchain: any[]) => ((index === 0) || blockchain[index].previousBlockId === blockchain[index - 1].id);
```

Comment:
a well named function contains a verb which describe what the function does.
Ex: const isTheBlockValid = () ...